examples/python/basic/build-xml.py

```diff
@@ -128,8 +124,8 @@
 third_tally.filters = [cell_filter, energy_filter, energyout_filter]
 third_tally.scores = ['scatter', 'nu-scatter', 'nu-fission']
 
-# Instantiate a TalliesFile, register all Tallies, and export to XML
-tallies_file = openmc.TalliesFile()
+# Instantiate a Tallies object, register all Tallies, and export to XML
```

Comment:
how about "Tallies collection" instead of "Tallies object" to mirror your description of the "Materials collection" above?